templates/engage/edge-month.html

```diff
@@ -0,0 +1,115 @@
+{% extends_with_args "engage/base_engage.html" with title="A guide to edge computing and the tools you need" meta_image="https://assets.ubuntu.com/v1/cdbfad95-Social+banner.jpg" meta_description="Join Canonical Product Managers for a series of webinars on edge computing to learn what is driving businesses to process their data at the edge and how you can add an edge computing layer to your business infrastructure. " %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1D_CuxaO_II0Jh9jRO9mYgRnO1J99s3DY-Iq78fPpvTc{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip p-strip--dark p-takeover--edge">
+  <div class="row u-equal-height">
+    <div class="col-8 u-vertically-center">
+      <div>
+      <h1>
+        Canonical presents:<br>Edge month
+      </h1>
+      <h4>
```

Comment:
Semantically, a `h4` element shouldn't directly follow a `h1`, if it needs to be a heading, a `h2` with `.p-heading--four` class will achieve the same thing. I think this could as well be a `p` element though(with that class applied).